brave-tests/src/main/java/brave/test/propagation/CurrentTraceContextTest.java

```diff
@@ -311,7 +311,7 @@ protected void is_inheritable(CurrentTraceContext inheritableCurrentTraceContext
   }
 
   @Test void restoresSpanAfterCallable() throws Exception {
-    try (Scope scope0 = currentTraceContext.newScope(context)) {
+    try (Scope scope = currentTraceContext.newScope(context)) {
```

Comment:
it was insane to use analysis to look for unused variables while this was called ws, scope, scope0, so forgive this change... but it really helped in the slog!